src/main/java/uk/gov/cslearning/catalogue/service/EventService.java

```diff
@@ -61,6 +67,21 @@ public Event save(String courseId, String moduleId, Event event){
                     String.format("Unable to find event: %s. Module does not exist: %s", eventId, moduleId));
         }
 
-        return module.getEvents().stream().filter(e -> e.getId().equals(eventId)).findFirst();
+        Optional<Event> result = module.getEvents().stream().filter(e -> e.getId().equals(eventId)).findFirst();
+
+        if(result.isPresent()){
+            Event event = result.get();
+
+            List<Booking> bookings = learnerRecordService.getEventBookings(eventId);
+
+            event.getVenue().setAvailability(event.getVenue().getCapacity());
+            bookings.forEach(b -> {
+                if (b.getStatus() != "Cancelled") {
```

Comment:
- We want only approved ones, there will be cancelled, declined, etc. 
- Never use the strings but the enums